src/battle_script_commands.c

```diff
@@ -3873,12 +3898,7 @@ static void Cmd_seteffectwithchance(void)
 {
     CMD_ARGS();
 
-    u32 percentChance;
-
-    if (GetBattlerAbility(gBattlerAttacker) == ABILITY_SERENE_GRACE)
-        percentChance = gBattleMoves[gCurrentMove].secondaryEffectChance * 2;
-    else
-        percentChance = gBattleMoves[gCurrentMove].secondaryEffectChance;
+    u32 percentChance = GetMoveSecondaryEffectChance(gBattlerAttacker, gBattleMoves[gCurrentMove].secondaryEffectChance);
```

Comment:
Maybe it would be cleaner to pass `gCurrentMove` instead of `secondaryEffectChance`.

Reply:
It would probably fit the function's own label better. I'll make the change.

Reply:
How about `CalcMoveSecondaryEffectChance`? To make it similar to `CalcMoveBasePower`.

Reply:
Sounds good to me. In the Discord server, Cancer Fairy and I settled for `CalculateMoveSecondaryEffectChance`, so we were all on pretty much the same wavelength, I'll rename and push for another review.